model_analyzer/config/generate/run_config_generator_factory.py

```diff
@@ -73,12 +73,17 @@ def create_run_config_generator(
         ensemble_composing_models = RunConfigGeneratorFactory._create_ensemble_composing_models(
             new_models, command_config, client, gpus)
 
-        if (command_config.run_config_search_mode == "quick"):
+        bls_composing_models = RunConfigGeneratorFactory._create_bls_composing_models(
+            command_config, client, gpus)
+
+        if (command_config.run_config_search_mode == "quick" or
+                bls_composing_models):
             return RunConfigGeneratorFactory._create_quick_plus_concurrency_sweep_run_config_generator(
                 command_config=command_config,
                 gpus=gpus,
                 models=new_models,
                 ensemble_composing_models=ensemble_composing_models,
+                bls_composing_models=bls_composing_models,
```

Comment:
I'm seeing ensemble and bls passed in separately everywhere. Are they different enough that you have to handle them separately?

Reply:
At this point they are - I would like to combine them, but my goal is to get the flow working first and then go back and make improvements if needed. (See the above comment (list vs. dict of list) as to why they are different.